resources/lang/en/admin/settings/general.php

```diff
@@ -174,7 +174,7 @@
     'saml_idp_metadata_help'    => 'You can specify the IdP metadata using a URL or XML file.',
     'saml_attr_mapping_username' => 'Attribute Mapping - Username',
     'saml_attr_mapping_username_help' => 'NameID will be used if attribute mapping is unspecified or invalid.',
-    'saml_forcelogin_label'     => 'SAML Force Login',
+    'saml_forcelogin_label'     => 'SAML Default Login',
```

Comment:
I have a small curiousity about the difference between the `saml_forcelogin_label` versus the `saml_forcelogin`, but not enough to block anything.